testutil/testnode/full_node.go

```diff
@@ -214,11 +215,34 @@ func DefaultNetwork(t *testing.T, blockTime time.Duration) (accounts []string, c
 		t.Log("tearing down testnode")
 		require.NoError(t, stopNode())
 		require.NoError(t, cleanupGRPC())
+		require.NoError(t, removeDir(path.Join([]string{tmCfg.RootDir, "config"}...)))
```

Comment:
[blocking]
Could you revert this change please? This is introduced in another PR